context/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id "be.insaneprogramming.gradle.animalsniffer" version "1.4.0"
+}
+
+description = 'gRPC: Context'
+
+dependencies {
+    compile libraries.guava,
```

Comment:
If this doesn't have a strict need for guava, it will be easier to reuse in other projects. can be a follow-up